### ISSUE TITLE
Difference with flymake-ruff

### ISSUE THREAD

**Issue by doolio** [cjfuller/ruff-flymake#1]
What is the difference of this package with `flymake-ruff` which you link to in the README. IIUC, `flymake-ruff` is also not intended to be used with a LSP client. 

In any case, could one not set `python-flymake-command` to use `ruff` instead of the default `pyflakes` and thus make this and `flymake-ruff` unnecessary? I've tried setting that variable accordingly but I can't get it to work.

Thanks for your time.

**Comment by cjfuller** [comment]
Hi! There's not really any intended difference here or reason to use this one. I started writing this when the one linked in the readme hadn't yet been added to MELPA; the linked one is more complete and works better. I just left this repo around for my own personal use.

> In any case, could one not set `python-flymake-command` to use ruff instead of the default `pyflakes` and thus make this and `flymake-ruff` unnecessary?

I have some vague recollection that this was the first thing I tried, and it didn't work or I ended up not wanting to do that, but I don't recall why at this point. (Some (unchecked) possibilities off the top of my head: incompatible error output format? Wanting to run both a different backend and ruff at the same time? Issues finding the correct path to ruff / virtualenv?)
